lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -0,0 +1,14 @@
+module ShopifyAPI
+  class FulfillmentOrder < Base
+    def self.all(options = {})
```

Comment:
With the `fulfillment_orders` method implemented on the order object, is this method necessary?

Reply:
That's right it can be queried on the order object. Any activeresource type will attempt to make queries, i.e. `FulfillmentOrder.all` will attempt to make a request even if this override method isn't included. Adding this just does the expected thing by routing to the order's implementation rather than failing without good diagnostics.